src/blog/lagom-1-3-9.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Lagom 1.3.9 is released!
+date: 2017-10-05
+author_github: rcavalcanti
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.3.9
+---
+
+The Lagom team has released Lagom 1.3.9. For a list of the specific changes in Lagom 1.3.9, see the [change log](/changelog.html).
+
+This release provides a few bug fixes and minor improvements. As an overview, this release contains the following:
+
+- a few dependencies updates (Play 2.5.17, Sbt 0.13.16 and ZooKeeper 3.4.10)
+- simplified actor system termination and JVM shutdown [978](https://github.com/lagom/lagom/issues/978) 
+- a fix for a bug preventing the usage of MariaDB JDBC driver [1024](https://github.com/lagom/lagom/issues/1024)
+- look up a migration using only the class name instead of the manifest containing the version number [952](https://github.com/lagom/lagom/issues/952
```

Comment:
I would expand on the last three:

```
 - the actor system termination when the cause of termination is a split brain has been simplified. This wil lmake the code more maintainable but it has no impact on how Lagom behaves.
 - we've had a great community contribution fixing a bug that makes Lagom compatible with MariaDB 
 - also from the community, we've had a contribution improving the persistent entities [Schema Evolution](https://www.lagomframework.com/documentation/1.3.x/java/Serialization.html#Schema-Evolution). This improvement only affects javadsl users.
```

Also, on the last item of the list there's a missing parenthesis on the link to the 952 issue.